src/tools/annotation/ArrowAnnotateTool.js

```diff
@@ -91,10 +91,6 @@ export default class ArrowAnnotateTool extends BaseAnnotationTool {
     );
   }
 
-  updateCachedStats(image, element, data) {
```

Comment:
I think this is fine, but need to verify it's not called generically from somewhere